### PR TITLE
docs: add ttl control section and faq

### PR DESCRIPTION
**Pull request by rubensfig**
Signed-off-by: Rubens Figueiredo <rubens.figueiredo@bisdn.de>

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -6,3 +6,6 @@ nav_order: 19
 
 1. Why do i get a latency above 1ms when trying to ping the IP address assigned to a switch port from a directly connected server ?
    - This is the delay incurred by using the OpenFlow interface on-switch. The traffic destined to the controller is encapsulated in the OpenFlow PACKET_IN, destined to the tap interface created by basebxd. This extra encapsulation and transmission over the OpenFlow channel results in a ~1.5ms delay when pinging the switch port. 
+
+2. Why do some packets with TTL=1 not reach my router?
+  - Packets traversing the switch may have a rule in Unicast Routing table to route the packets to the CPU. In the `eBGP` the packets carrying BGP messages have a TTL=1, which results in the packet being dropped when they would reach the CPU. For this reason there exists a switch control `L3UcastTtl1ToCpu` to enable routing TTL=1 packets to the CPU. For more information refer to [TTL control section](setup/setup_standalone.md#ttl-controls).
```
Comment (jklare) [inline faq.md]:
```suggestion
  - Switches that forward packtes may have a rule in the "Unicast Routing" table to route the packets to the CPU. In `eBGP` the packets carrying BGP messages have a TTL=1, which results in the packet being dropped when they would reach the CPU. For this reason there exists a switch control `L3UcastTtl1ToCpu` to enable routing TTL=1 packets to the CPU. For more information refer to [TTL control section](setup/setup_standalone.md#ttl-controls). 
```

File: setup/setup_standalone.md [hunk 2]
```diff
@@ -256,6 +256,25 @@ exit
...
+```
+
+To enable routing packets with TTL=1 to CPU. 
+The following command
+
+```
+client_drivshell SwitchControl L3UcastTtl1ToCpu=1
+```
+
+Will activate the same behaviour on the switch in a non-persistent way. By omitting the assignment the status is reported.
```
Comment (jklare) [inline setup/setup_standalone.md]:
```suggestion
Will activate the same behaviour on the switch in a non-persistent way.
To print the currently set value, you can run:

```
client_drivshell SwitchControl L3UcastTtl1ToCpu
```
```

File: setup/setup_standalone.md [hunk 3]
```diff
@@ -256,6 +256,27 @@ exit
...
+
+```
+client_drivshell SwitchControl L3UcastTtl1ToCpu=1
+```
+
+Will activate the same behaviour on the switch in a non-persistent way.
+To print the currently set value, you can run:
+
+
 ## Bundled software with BISDN Linux
```
Comment (rubensfig) [inline setup/setup_standalone.md]:
```suggestion

client_drivshell SwitchControl L3UcastTtl1ToCpu=1
``` 
```

File: setup/setup_standalone.md [hunk 4]
```diff
@@ -256,6 +256,30 @@ exit
 
 Note the absence of client_drivshell and the single exit statement at the end.
 
+### TTL controls
+
+BISDN Linux has a default configuration file under `/etc/ofdpa/rc.soc` controlling both the Broadcom SDK and OF-DPA.
+
+The file has the following control by default,
```
Comment (jklare) [inline setup/setup_standalone.md]:
```suggestion
To enable routing packets with TTL=1 to CPU by default, the file contains the following setting:
```

File: setup/setup_standalone.md [hunk 5]
```diff
@@ -256,6 +256,30 @@ exit
...
+
+```
+SwitchControl L3UcastTtl1ToCpu=1
+```
+
+To enable routing packets with TTL=1 to CPU.
```
Comment (jklare) [inline setup/setup_standalone.md]:
```suggestion
```

File: setup/setup_standalone.md [hunk 6]
```diff
@@ -256,6 +256,30 @@ exit
...
+BISDN Linux has a default configuration file under `/etc/ofdpa/rc.soc` controlling both the Broadcom SDK and OF-DPA.
+
+The file has the following control by default,
+
+```
+SwitchControl L3UcastTtl1ToCpu=1
+```
+
+To enable routing packets with TTL=1 to CPU. 
+The following command
```
Comment (jklare) [inline setup/setup_standalone.md]:
```suggestion
The following command will activate the same behaviour on the switch in a non-persistent way:
```

File: setup/setup_standalone.md [hunk 7]
```diff
@@ -256,6 +256,30 @@ exit
...
+
+```
+client_drivshell SwitchControl L3UcastTtl1ToCpu=1
+```
+
+Will activate the same behaviour on the switch in a non-persistent way.
```
Comment (jklare) [inline setup/setup_standalone.md]:
```suggestion
```

**Pull request merged**